Gemfile

```diff
@@ -18,7 +19,8 @@ gem 'nilify_blanks'
 gem 'nokogiri'
 gem 'pg'
 gem 'pg_search'
-gem 'rails', '3.2.22.5'
+gem 'protected_attributes'
```

Comment:
Means we can use "attr_accessible" and "attr_protected" now - they've been removed from rails by default with 4.0 and it also now requires strong params (meaning if you don't do anything in the controller, it will crash with an error, since the params are not whitelisted). Alternatively this thing makes it optionally done in the models with attr_accessible etc, which I've left in place for replies and posts and some other models.